bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/EchoHandler.java

```diff
@@ -739,7 +778,8 @@ private void startTextToSpeech(Connection connection, Device device, String text
             this.ignoreVolumeChange = scheduler.schedule(this::stopIgnoreVolumeChange, 2000, TimeUnit.MILLISECONDS);
         }
         if (text.startsWith("<speak>") && text.endsWith("</speak>")) {
-            connection.sendAnnouncement(device, text, null, null, textToSpeechVolume, lastKnownVolume);
+            String plainText = text.replaceAll("<[^>]+>", "");
```

Comment:
While this is the same as before, I have a question here: are you sure it is correct in all cases? What is the text contains `<something>`, should this be removed?

Reply:
This change is ok. In the case if the text starts with `<speak>` and end with `</speak>`, the user uses SSML which is XML. This means, if the text should contains `<something>` it must be escaped to `&lt;something&gt;`. All xml text must be removed for the body value, because this is shown in the display of the device.